test/e2e/testing-manifests/storage-csi/external-attacher/rbac.yaml

```diff
@@ -79,3 +79,4 @@ subjects:
 roleRef:
   kind: Role
   name: external-attacher-cfg
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
These files are conceptually read-only copies of the upstream files. If you find that they need to be updated, please also submit upstream or at least file bugs.

In this case we are fine because the upstream files were fixed independently.

Reply:
yes exactly.  Also eventually these test manifests should pull directly from upstream so we don't have to keep syncing them

Reply:
I was thinking about that the other day when changing the docs to pull from upstream. It wouldn't be hard to add support for loading the data via http(s), but do we want that (tests self-contained vs. depending on a growing number of external services), and how much do we trust https:/raw.githubusercontent.com to give us the right content?

It's probably okay, I just wanted to raise awareness about that aspect.